usbl_evologics.orogen

```diff
@@ -35,6 +35,8 @@ task_context "Task" do
    # Set interface with device. SERIAL or ETHERNET.
    # The interface type may be different from the one set by io_port, in case there is an intermediary, i.e. sys_mgm.
    property "interface", "/usbl_evologics/InterfaceType", :SERIAL
+   # Granularity (updateHook() minimum interval in ms, default: 500ms)
+   property "granularity", "uint32_t", 500
```

Comment:
There is an example [here](https://github.com/Brazilian-Institute-of-Robotics/drivers-orogen-usbl_evologics/blob/c4d4702d775cd8b2013693b336b0d7fa92415c0d/tasks/Task.cpp#L38).
I'd prefer to set in the .orogen file, but I don't know if it's possible.